### DISCUSSION TITLE
When will pubpub be available for stable self hosting with good documentation.

### DISCUSSION THREAD

**Discussion by cosmobird** [knowledgefutures/pubpub#3077] [Support]
Hi community, since i came to know about pubpub, its ability, its features, and documenting capacity, i have always been longing for a version that has long term support with self hosting procedure and documentation. Probably a container image would also be helpful to test before going for production.

I tried to do it as per the github instructions, but fails.

**Comment by cfcohen** [comment]
I too am interested in self-hosting, just for personal use.  When I asked several years ago, I was told it wasn't a priority.   Since Pubpub has recently announced "PubPub Legacy" this has become a more urgent issue for me.   Has anyone other the developers gotten this open source project to actually run?

**Comment by smmpanel01a** [comment]
I like how this didn’t try to overcomplicate things. You stuck to a natural, friendly tone and made the content approachable. It’s small details like that which keep readers around. [smm service supplier
](https://smmpanel.com/)

**Comment by ipf2014** [comment]
It’s great to see others interested in self-hosting PubPub especially with the announcement of "PubPub Legacy," long-term sustainability becomes a key concern. A containerized version or official Docker image would definitely simplify *[*local testin**g](https://ipf2014.de/) and deployment. Given the complexity of the current GitHub setup and limited documentation, it would be incredibly helpful if the developers or community members who’ve had success could share a reproducible walkthrough or at least a working configuration. Even basic guidance on required environment variables or dependency versions could make a big difference for those of us aiming to run it independently.
